lib/Service/OpenAiAPIService.php

```diff
@@ -182,15 +327,27 @@ public function transcribeFile(?string $userId, File $file, bool $translate = fa
 	 */
 	public function transcribe(?string $userId, string $audioFileContent, bool $translate = true,
 							   string $model = Application::DEFAULT_TRANSCRIPTION_MODEL_ID): array {
+		if($this->isQuotaExceeded($userId, Application::QUOTA_TYPE_TRANSCRIPTION)){
+			return ['error' => 'Transcription usage quota exceeded'];
```

Comment:
Aaah

Reply:
Fixed in: https://github.com/nextcloud/integration_openai/commit/6697be3b38618938a3fe9e5ab264d1eb7aeec0fe